### DISCUSSION TITLE
Hue-saturation box component

### DISCUSSION THREAD

**Discussion by Fhuyu** [jaames/iro.js#171] [Feature Requests]
Hello! 
I'm looking to enable hue colors in the box layout, but I can't find the option.
Is it possible?
Expected result : 
![image](https://user-images.githubusercontent.com/56589674/85055219-822a1800-b19d-11ea-8a2f-6ae22a92741e.png)
Thank you in advance for your answer!

**Comment by jaames** [comment]
At the moment, this isn't possible - you'll want a hue slider with the box instead.

Personally I don't really like this type of color picker because I find them unintuitive (and that kindof goes against what iro.js is trying to be - intuitive)... but I suppose if more people want it I could stop being stubborn and add it as an option. :P
